composer.json

```diff
@@ -12,7 +12,7 @@
     "consolidation/robo": "^1.0.6",
     "guzzlehttp/guzzle": "^6.2",
     "psy/psysh": "^0.8",
-    "symfony/console": "^3.2",
+    "symfony/console": "^3.3",
```

Comment:
It's generally not necessary to pin to a specific minor or patch version unless there is something in your code that depends on a bugfix or feature introduced in that version. ^3.2 will also select 3.3. That said, neither is there any harm in the practice.